README.org

```diff
@@ -209,3 +228,13 @@ Please support one of:
 *** Projects
 
 -  [[https://github.com/emacs-helm/helm][helm]]
+
+
+* COMMENT Config
+:PROPERTIES:
+:TOC:      ignore
+:END:
+
+# Local Variables:
+# before-save-hook: org-make-toc
```

Comment:
I am being told `Lisp error: (void-function org-make-toc)`. Could you add some instructions to the commit message pease?

Reply:
You need to have the package `org-make-toc` installed.  Maybe you don't want to use it, but if you want a table of contents visible on GitHub, it's necessary, because the built-in org-export ToC only applies when exported.